modules/creating-scaling-machine-sets-compute-nodes-networking.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// IPI
+// * installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+// UPI
+// * installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-scaling-machine-sets-compute-nodes-networking.adoc_{context}"]
+= Optional: Scaling each machine set to compute nodes
+
+To apply a customized `br-ex` bridge configuration to all compute nodes in your {product-title} cluster, you must edit your `MachineConfig` custom resource definition (CRD) and modify its roles. Additionally, you must create a `BareMetalHost` CRD that defines information for your bare-metal machine, such as hostname, credentials, and so on.
+
+After you configure these resources, you must scale machine sets, so that the machine sets can apply the resource configuration to each compute node and reboot the nodes.
+
+.Prerequisites
+
+* You created a `MachineConfig` manifest object that includes a customized `br-ex` bridge configuration.
+
+.Procedure
+
+. Enter the following command to edit the `MachineConfig` CRD:
++
+[source,terminal]
+----
+$ oc edit mc <machine_config_CRD_name>
+----
+
+. Add each compute node configuration to the CRD, so that the CRD can manage roles for each defined compute node in your cluster.
+
+. Create an `extraworker-secret` `Secret` object that has a minimal static IP configuration.
+
+. Apply the `extraworker-secret` `Secret` object to each node in your cluster by entering the following command. This step provides each compute node access to the Ignition config file.
++
+[source,terminal]
+----
+$ oc apply -f ./extraworker-secret.yaml
+----
+
+. Create a `BareMetalHost` resource and specify the network secret in the `preprovisioningNetworkDataName` parameter.
++
+.Example `BareMetalHost` resource with an attached network secret
+[source,yaml]
+----
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+spec:
+# ...
+  preprovisioningNetworkDataName: ostest-extraworker-0-network-config-secret
+# ...
+----
+
+. To mange the `BareMetalHost` object within the `openshift-machine-api` namespace of your cluster, enter the following command:
```

Comment:
Minor nit, maybe more specifically/accurately:
```suggestion
. To mange the `BareMetalHost` object within the `openshift-machine-api` namespace of your cluster, change to the namespace by entering the following command:
```
Since this command itself isn't actually "managing the `BareMetalHost` object" technically. Tho I'm splitting hairs. 🙃

Reply:
:heavy_check_mark: